app/controllers/chatrooms_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ChatroomsController < ApplicationController
+  before_action :authenticate_user!
+  def index
+    @user = User.find(params[:user_id])
```

Comment:
you don't have to do this manually. check the documentation of `devise` for getting the current logged in user.